index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tribute Page</title>
+
```

Comment:
Extra spaces should be removed from the code

Reply:
Please remember to include why you suggest a change. Some things might be obvious to you but not to someone else. :+1: :smiley: :smiley:

Reply:
- [ ] As Santiago suggested, it would be great if you edit this comment to include an explanation of the requested change. Why extra spaces should be removed from the code? How removing extra spaces can improve the code base?

💡 Hint:
> * This is a well-explained request: _Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible_
>
> * This request lacks a "why": _Why don't you use an article tag here instead of a div?_

Please, consider applying this change! 🤓 🛠️